Assets/Plugins/CountlySDK/Enums/FeaturesEnum.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Plugins.CountlySDK.Enums
 {
-    internal enum FeaturesEnum
+    public enum FeaturesEnum
```

Comment:
Rename this enum to "Features"

Reply:
WELL, this belongs to consents it will update all files related to consents.
but I have completed it.

Reply:
that's fine